docker-compose.yml

```diff
@@ -48,11 +36,11 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
-  probation-offender-search:
```

Comment:
I've moved one of the existing tests from calling probation indexer to calling prison indexer